tutorials/node.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Hello from Node
+parent: Tutorials
+nav_order: 1
+---
+
+# Hello Decentralized World -- from Node
+
+## Quickstart
+To show how easy Tupelo is to use we are going to create our first ChainTree and then add
+data to it ("Hello World" of course) and have it signed by the Tupelo TestNet.  
+
+We are going to accomplish this by using the Tupelo WASM SDK and a node repl session.
+
+First you will need to use [npm](https://www.npmjs.com/get-npm) to install two dependencies.
+
+```
+npm install tupelo-wasm-sdk@0.6.0-rc3
+npm install multicodec@0.5.6
```

Comment:
seems odd that this isn't explicitly installed with the sdk, I'll look into it

Reply:
It was that the latest multicodec broke things.  May be fixed by now.